File: Multiples Of Three and Five/MultiplesOf3And5.java
public class MultiplesOf3And5 {
    public static void main(String[] args) {
        System.out.println(multiplesOf3and5(10));
    }
    public static long multiplesOf3and5(int number) {
        long sum = 0;
        for (int i = 0; i < number; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                sum += i;
            }
        }

        return sum;
    }
}

File: 001: Multiples Of Three and Five/Multiplesof3And5.py
def sum_multiples_3_5(num):
    sum = 0
    for i in range(num):
        if (i % 3 == 0 or i % 5 == 0):
            sum += i

    return sum


def main():
    num = int(input("Enter a number: "))
    print(sum_multiples_3_5(num))


main()
